nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -19,20 +19,28 @@
 import torch
 import torch.distributed
 import torch.nn as nn
+from omegaconf import ListConfig
 
+from nemo.collections.asr.models.configs import FramewiseStreamingConfig
+from nemo.collections.asr.parts.mixins.streaming import StreamingEncoder
+from nemo.collections.asr.parts.submodules.causal_convs import CausalConv1D
 from nemo.collections.asr.parts.submodules.conformer_modules import ConformerLayer
-from nemo.collections.asr.parts.submodules.multi_head_attention import PositionalEncoding, RelPositionalEncoding
+from nemo.collections.asr.parts.submodules.multi_head_attention import (
+    MultiHeadAttention,
+    PositionalEncoding,
+    RelPositionalEncoding,
+)
 from nemo.collections.asr.parts.submodules.subsampling import ConvSubsampling, StackingSubsampling
 from nemo.core.classes.common import typecheck
 from nemo.core.classes.exportable import Exportable
 from nemo.core.classes.mixins import adapter_mixins
 from nemo.core.classes.module import NeuralModule
-from nemo.core.neural_types import AcousticEncodedRepresentation, LengthsType, NeuralType, SpectrogramType
+from nemo.core.neural_types import AcousticEncodedRepresentation, ChannelType, LengthsType, NeuralType, SpectrogramType
 
 __all__ = ['ConformerEncoder']
 
 
-class ConformerEncoder(NeuralModule, Exportable):
+class ConformerEncoder(NeuralModule, Exportable, StreamingEncoder):
```

Comment:
Pedantic, move exportable to last in the list.

Reply:
done.